ts/features/pn/store/types/types.ts

```diff
@@ -1,27 +1,10 @@
 import * as t from "io-ts";
-import { enumType } from "italia-ts-commons/lib/types";
 import { ServiceId } from "../../../../../definitions/backend/ServiceId";
 import { MvlAttachment } from "../../../mvl/types/mvlData";
 import { UTCISODateFromString } from "../../utils/dates";
 
-export enum NotificationStatusEnum {
-  "IN_VALIDATION" = "IN_VALIDATION",
-  "ACCEPTED" = "ACCEPTED",
-  "DELIVERING" = "DELIVERING",
-  "DELIVERED" = "DELIVERED",
-  "VIEWED" = "VIEWED",
-  "EFFECTIVE_DATE" = "EFFECTIVE_DATE",
-  "PAID" = "PAID",
-  "UNREACHABLE" = "UNREACHABLE",
-  "REFUSED" = "REFUSED",
-  "CANCELLED" = "CANCELLED"
-}
-
 export type NotificationStatus = t.TypeOf<typeof NotificationStatus>;
-export const NotificationStatus = enumType<NotificationStatusEnum>(
```

Comment:
another option would be to create a custom io-ts decoder that composes `NotificationStatus` with `t.String` and decodes an `Either<String, NotificationStatus>`. If the keys is a valid one from the enum, it will be decoded as `Right< NotificationStatus>`, if it's not it will be decoded as `Left<String>`.
Best of both worlds. 😄